### DISCUSSION TITLE
How to parse Hash/Fragment in Url

### DISCUSSION THREAD

**Discussion by alexbot89** [fastify/fastify#3796] [Q&A] [answered]
I have a route that gets called via GET by a server and this server is appending a hash/fragment like so.

http://localhost/myroute?id=05c94e32-bed0-478e-8ac2-df127b4a079d#fragmentid=6a1c08ae-1344-415d-82ca-cfafd33b750f

I attempted to access this value after the `#` with:

`request.url` and `request.query` but only got:
- http://localhost/myroute?id=05c94e32-bed0-478e-8ac2-df127b4a079d
- { id: 05c94e32-bed0-478e-8ac2-df127b4a079d }

respectively.

Is there a way in fastify to access `#fragmentid`?

**Comment by jsumners** [comment, marked as answer]
Servers never receive the URI fragment. That is purely for clients to interpret https://www.w3.org/DesignIssues/Fragment.html
